### ISSUE TITLE
USER ISSUE: Caught exception invoking RPC PlayerInteract on Player

### ISSUE THREAD

**Issue by Krayt** [StrangeLoopGames/EcoIssues#6292]
**Version:** 0.7.2.3 beta

**Steps to Reproduce:**
I tried to open a chest after I levelled up "strong back" and "calorie consumption" skill. I don't know how to reproduce the error.

**Expected behavior:**
No Exception should have been thrown

**Actual behavior:**
The following error message popped up:
```
Caught exception:

<size=60,00%>Caught exception invoking RPC PlayerInteract on Player!

Value cannot be null.
Parameter name: item


Stack:
  at System.Collections.Immutable.ImmutableHashSet`1[T].Contains (T item) [0x00000] in <bffdd0b30ee64b3f9ebc6a9f353a97c1>:0 
  at Eco.Core.Utils.ThreadSafeHashSet`1[T].Contains (T item) [0x00000] in <9dd4d243d48e4c0cb5bb8fd00ab12543>:0 
  at Eco.Core.Controller.ControllerInstanceInfo.TryBind (Eco.Shared.Networking.INetClient client) [0x00006] in <9dd4d243d48e4c0cb5bb8fd00ab12543>:0 
  at Eco.Core.Controller.ControllerManager.TryBindController (Eco.Core.Controller.IController controller, Eco.Shared.Networking.INetClient boundClient) [0x00054] in <9dd4d243d48e4c0cb5bb8fd00ab12543>:0 
  at Eco.Core.Controller.ControllerManager.PackageController (Eco.Core.Controller.IController controller, Eco.Shared.Networking.INetClient boundClient) [0x00000] in <9dd4d243d48e4c0cb5bb8fd00ab12543>:0 
  at Eco.Gameplay.Objects.WorldObject.GetBSON (Eco.Gameplay.Interactions.InteractionContext context) [0x000a1] in <5a6fe3953ad54612b7e68da0f55f09a8>:0 
  at Eco.Gameplay.Objects.WorldObject.OnActInteract (Eco.Gameplay.Interactions.InteractionContext context) [0x0005e] in <5a6fe3953ad54612b7e68da0f55f09a8>:0 
  at Eco.Gameplay.Players.Player+<>c__DisplayClass45_0.<PlayerInteract>b__2 (Eco.Gameplay.Interactions.IInteractable inter) [0x00000] in <5a6fe3953ad54612b7e68da0f55f09a8>:0 
  at Eco.Gameplay.Interactions.InteractionExtensions+<>c__DisplayClass0_1`1[T].<ExecuteUntilSuccess>b__1 () [0x00000] in <5a6fe3953ad54612b7e68da0f55f09a8>:0 
  at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess (System.Collections.Generic.IEnumerable`1[T] results) [0x00016] in <5a6fe3953ad54612b7e68da0f55f09a8>:0 
  at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess[T] (System.Collections.Generic.IEnumerable`1[T] enumerable, System.Func`2[T,TResult] resultFunc) [0x0001f] in <5a6fe3953ad54612b7e68da0f55f09a8>:0 
  at Eco.Gameplay.Players.Player.PlayerInteract (Eco.Shared.Items.InteractionInfo info) [0x000df] in <5a6fe3953ad54612b7e68da0f55f09a8>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in <f56c876907e742b0aa586f051fcce845>:0 



Stack:
  at Eco.Shared.Networking.RPCManager.TryInvoke (System.Object controller, System.String methodname, Eco.Shared.Serialization.BSONObject bsonArgs, System.Object& result) [0x00060] in <87b449fa3be548e4a9bca65d2ad34821>:0 
  at Eco.Shared.Networking.RPCManager.InvokeOn (Eco.Shared.Serialization.BSONObject bson, System.Object controller, System.String methodname) [0x00055] in <87b449fa3be548e4a9bca65d2ad34821>:0 
  at Eco.Shared.Networking.RPCManager.HandleReceiveRPC (Eco.Shared.Networking.INetClient client, Eco.Shared.Serialization.BSONObject bson) [0x0003d] in <87b449fa3be548e4a9bca65d2ad34821>:0 
  at Eco.Plugins.Networking.Clients.ClientManager.ReceiveEvent (Eco.Shared.Networking.INetClient client, Eco.Shared.Networking.NetworkEvent netEvent, Eco.Shared.Serialization.BSONObject bson) [0x00021] in <cb46936a53a64832a4f16709d80dbff3>:0 
  at Eco.Shared.Networking.NetObject.ReceiveEvent (Eco.Shared.Networking.INetClient client, Eco.Shared.Networking.NetworkEvent netEvent, Eco.Shared.Serialization.BSONObject bson) [0x00000] in <87b449fa3be548e4a9bca65d2ad34821>:0 
  at Eco.Shared.Networking.NetPort.ProcessBuffer () [0x00016] in <87b449fa3be548e4a9bca65d2ad34821>:0 
  at Eco.Shared.Networking.NetPort.Update () [0x0000a] in <87b449fa3be548e4a9bca65d2ad34821>:0 
  at Eco.Plugins.Networking.Clients.Client.<.ctor>b__56_0 () [0x00012] in <cb46936a53a64832a4f16709d80dbff3>:0 
</size>
```

**Comment by MarjieAVolk** [comment]
Related to https://github.com/StrangeLoopGames/EcoIssues/issues/5804

**Comment by fisele** [comment]
#6156

**Comment by metachronism** [comment]
fixed 7.3
